wurst/objects/items/ScrollsDefinition.wurst

```diff
@@ -0,0 +1,421 @@
+            + makeToolTipCooldown(CYCLONE_COOLDOWN)
+
+
+// Entangling Root effect differs from being casted to trolls or animals
+// Trolls : Cast ensnare
+// Animals : Cast Entangling Root
+@compiletime function createEntaglingRootAbility()
+    new AbilityDefinitionEnsnareCreep(ABILITY_ROOT_TROLL)
+        ..setDummyAbility()
+        ..setBuffs(1, commaList(BUFF_ROOT_GROUND, BUFF_ROOT_AIR))
+        ..setMissileArt("")
+        ..setDurationHero(1, ROOT_DURATION_HERO)
+        ..setDurationNormal(1, ROOT_DURATION_NORMAL)
+        ..setTargetsAllowed(1, commaList(
+            TargetsAllowed.enemies,
+            TargetsAllowed.ground,
+            TargetsAllowed.neutral,
+            TargetsAllowed.organic,
+            TargetsAllowed.hero
+            ))
+        ..setName("Entangling Root Troll")
+        ..setEditorSuffix("(Wurst)")
+
+    new AbilityDefinitionEntanglingRootscreep(ABILITY_ROOT_HOSTILE)
+        ..setDummyAbility()
+        ..setCastRange(1, 99999)
+        ..setDamageperSecond(1, ROOT_DMG)
+        ..setDurationHero(1, ROOT_DURATION_HERO)
+        ..setDurationNormal(1, ROOT_DURATION_NORMAL)
+        ..presetTargetsAllowed(lvl -> commaList(
+            TargetsAllowed.ground,
+            TargetsAllowed.enemies,
+            TargetsAllowed.neutral,
+            TargetsAllowed.organic
+            ))
+        ..setName("Entangling Root Hostile")
+        ..setEditorSuffix("(Wurst)")
+
+    createItemChannelCast(ABILITY_ROOT_CAST)
+        ..setCooldown(1, ROOT_COOLDOWN)
+        ..setCastRange(1, ROOT_CAST_RANGE)
+        ..presetTargetTypes(Targettype.UNIT)
+        ..setTargetsAllowed(1, commaList(
+            TargetsAllowed.ground,
+            TargetsAllowed.enemies,
+            TargetsAllowed.organic
+        ))
+        ..setName("Root Item cast")
+        ..setManaCost(1, 0)
+
+
+@compiletime function createFireBallAbility() returns AbilityDefinitionFireBoltwarlock
+    return new AbilityDefinitionFireBoltwarlock(ABILITY_FIREBOLT)
+        ..setName("Scroll of Fire Ball")
+        ..setDamage(1, FIRE_DMG)
+        ..setMissileArt(Abilities.phoenix_Missile)
+        ..setCastRange(1, FIRE_CAST_RANGE)
+        ..setManaCost(1, 0)
+        ..setDurationHero(1, FIRE_DURATION_HERO)
+        ..setDurationNormal(1, FIRE_DURATION_NORMAL)
+        ..setCooldown(1, FIRE_COOLDOWN)
+        ..setEditorSuffix("(Wurst)")
+
+// Create 2 skeleton warrior following the caster
+// Skeleton warrior got the locust ability, meaning they can't be targetted/attacked and randomly move around, attacking surrounding units
+@compiletime function createSkeletalMinionAbility() returns AbilityDefinitionCryptLordLocustSwarm
+    return new AbilityDefinitionCryptLordLocustSwarm(ABILITY_SKELETAL_MINION)
+        ..setName("Scroll of Living Dead")
+        ..setCheckDependencies(false)
+        ..setDamageReturnFactor(1, 0)
+        ..setNumberofSwarmUnits(1, 2)
+        ..setDamageReturnThreshold(1, 600)
+        ..setMaxSwarmUnitsPerTarget(1, 5)
```

Comment:
Is this maximum tracked across multiple scrolls? If three trolls each use a scroll next to the same enemy, can that enemy be focused by all 6 skeletons?

Reply:
> Is this maximum tracked across multiple scrolls?
Yes, enemy can be focused by 5 skeleton max.